### ISSUE TITLE
False positive on @ rule: @-o-viewport

### ISSUE THREAD

**Issue by kweij** [CSSLint/csslint#561]
The following code:
```css
@-o-viewport {
    width: device-width;
}
```
will unexpectedly raise the following error: `Unknown @ rule: @-o-viewport`

This @ rule is however valid and should not raise an error.

**Comment by aaronlademann-wf** [comment]
Is there a workaround for this?

**Comment by replete** [comment]
I'm getting this error.

**Comment by kweij** [comment]
`Approximate duplicate of #443`

Indeed, this is a duplicate of the mentioned issue. In that issue the following is stated:

`@replete Right, it's fixed in master but the maintainers haven't made a release version since then; see #514.`

The release of this fix is planned for the [1.0.0](https://github.com/CSSLint/csslint/milestone/8) milestone which is not due yet. However, since this issue is apparently fixed in master, I'll go ahead an close this issue.
